multiversx_sdk_cli/tests/test_cli_transactions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+from typing import Any
+
+from multiversx_sdk_cli.cli import main
+
+testdata_path = Path(__file__).parent / "testdata"
+
+
+def test_relayed_v1_transaction(capsys: Any):
+    multi_user_pem = testdata_path / "multiple_addresses.pem"
+    address_index = 1
+
+    return_code = main([
+        "tx", "new",
+        "--pem", str(multi_user_pem),
+        "--pem-index", str(address_index),
+        "--receiver", "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u",
+        "--nonce", "198",
+        "--gas-limit", "60000000",
+        "--data", "getContractConfig",
+        "--version", "1",
+        "--chain", "T",
+        "--relay"
```

Comment:
In the future, we should add support for relayed V2, as well.

Reply:
Indeed